src/overlays/actors/ovl_En_Bjt/z_en_bjt.c

```diff
@@ -31,49 +53,408 @@ const ActorInit En_Bjt_InitVars = {
     (ActorFunc)EnBjt_Draw,
 };
 
-// static ColliderCylinderInit sCylinderInit = {
-static ColliderCylinderInit D_80BFDF48 = {
-    { COLTYPE_HIT1, AT_NONE, AC_NONE, OC1_ON | OC1_TYPE_ALL, OC2_TYPE_1, COLSHAPE_CYLINDER, },
-    { ELEMTYPE_UNK1, { 0x00000000, 0x00, 0x00 }, { 0x00000000, 0x00, 0x00 }, TOUCH_NONE | TOUCH_SFX_NORMAL, BUMP_NONE, OCELEM_ON, },
+static ColliderCylinderInit sCylinderInit = {
+    {
+        COLTYPE_HIT1,
+        AT_NONE,
+        AC_NONE,
+        OC1_ON | OC1_TYPE_ALL,
+        OC2_TYPE_1,
+        COLSHAPE_CYLINDER,
+    },
+    {
+        ELEMTYPE_UNK1,
+        { 0x00000000, 0x00, 0x00 },
+        { 0x00000000, 0x00, 0x00 },
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
+        BUMP_NONE,
+        OCELEM_ON,
+    },
     { 10, 68, 0, { 0, 0, 0 } },
 };
 
-// sColChkInfoInit
-static CollisionCheckInfoInit2 D_80BFDF74 = { 0, 0, 0, 0, MASS_IMMOVABLE };
+static CollisionCheckInfoInit2 sColChkInfoInit = { 0, 0, 0, 0, MASS_IMMOVABLE };
+
+typedef enum {
+    /* -1 */ TOILET_HAND_ANIM_NONE = -1,
+    /*  0 */ TOILET_HAND_ANIM_WAITING,
+    /*  1 */ TOILET_HAND_ANIM_WAITING_MORPH,  // Same as TOILET_HAND_ANIM_WAITING, but with morph frames
+    /*  2 */ TOILET_HAND_ANIM_WAGGING_FINGER, // Wrong
+    /*  3 */ TOILET_HAND_ANIM_THUMBS_UP,      // Right
+    /*  4 */ TOILET_HAND_ANIM_OPEN_HAND,
+    /*  5 */ TOILET_HAND_ANIM_FIST // i.e. holding the reward
+} ToiletHandAnimations;
+
+static AnimationInfoS sAnimationInfos[] = {
+    /* 0 */ { &gToiletHandWaitingAnim, 1.0f, 0, -1, 0, 0 },
+    /* 1 */ { &gToiletHandWaitingAnim, 1.0f, 0, -1, 0, -4 },
+    /* 2 */ { &gToiletHandWaggingFingerAnim, 1.0f, 0, -1, 0, -4 },
+    /* 3 */ { &gToiletHandThumbsUpAnim, 1.0f, 0, -1, 0, -4 },
+    /* 4 */ { &gToiletHandOpenHandAnim, 1.0f, 0, -1, 0, -4 },
+    /* 5 */ { &gToiletHandFistAnim, 1.0f, 0, -1, 0, -4 },
+};
+
+void EnBjt_UpdateAnimation(EnBjt* this) {
```

Comment:
```suggestion
void EnBjt_UpdateSkelAnime(EnBjt* this) {
```
I think `SkelAnime_Update` does more than just update the animation.